integration-test-groups/cxf-soap/cxf-soap-server/pom.xml

```diff
@@ -136,6 +140,19 @@
                         </exclusion>
                     </exclusions>
                 </dependency>
+                <dependency>
+                    <groupId>org.apache.camel.quarkus</groupId>
```

Comment:
You'll need to have the `-deployment` dependencies ordered alphabetically for the CI checks to pass.

If you run `mvn process-resources -Pformat -N` from the root of the project, it should adjust it for you.

Reply:
Thanks, i thought so. I will sort it.

Reply:
`mvn process-resources -Pformat -N` doesn't work for me, because i don't want to build whole project :(

Reply:
I've got this error 
```
[ERROR] Failed to execute goal com.mycila:license-maven-plugin:4.5:format (license-format) on project camel-quarkus: Unable to find a comment style definition for some files. You may want to add a custom mapping for the relevant file extensions.
```